tests/test_integration_autorunner.py

```diff
@@ -49,11 +49,10 @@
 train_param = (
     {
         "CUDA_VISIBLE_DEVICES": list(range(num_gpus)),
-        "num_iterations": int(4 / num_gpus),
-        "num_iterations_per_validation": int(4 / num_gpus),
         "num_images_per_batch": 2,
-        "num_epochs": 1,
-        "num_warmup_iterations": int(4 / num_gpus),
+        "num_epochs": 2,
+        "num_epochs_per_validation": 1,
+        "num_warmup_epochs": 1,
```

Comment:
segresnet algo uses a key "warmup_epochs" for this, not "num_warmup_epochs", 

but this value is determined automatically, as 1% of the total data size, it will be 1 in this test case anyway.

Reply:
@myron thanks for pointing it out! How about unifying all of them to `num_warmup_epochs` in the config?

Reply:
I'm OK with the name, but I think we should go over all common parameters, and then unify them all together.

Reply:
Totally agree @myron . Thanks!

Reply:
`num_warmup_epochs` is originally for algorithms other than segresnet. I unified the numbers for all four algorithms in the template.